src/main/java/org/folio/qm/converter/ParsedRecordDtoToQuickMarcConverter.java

```diff
@@ -76,6 +80,8 @@ public QuickMarc convert(ParsedRecordDto parsedRecordDto) {
         .map(this::dataFieldToQuickMarcField)
         .collect(Collectors.toList()));
 
+      fields.sort(QUICK_MARC_FIELDS_COMPARATOR);
```

Comment:
Maybe we can use comparator inside stream? For example:
```
 fields.addAll(marcRecord.getDataFields()
        .stream()
        .sorted(Comparator.comparingInt(o -> Integer.parseInt(o.getTag())))
        .map(this::dataFieldToQuickMarcField)
        .collect(Collectors.toList()));
```

Reply:
There we have data fields, but also we have control fields. All of them should be added to resulted `fields` collection and all of these fields should be sorted. That's why I put a sorting statement after both types of fields are added to the collection.